products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -368,27 +350,6 @@ public struct CommandExtendedResult {
 
 }
 
-/// The BuildKey encodes the key space used by the BuildSystem when using the
-/// core BuildEngine.
-public struct BuildKey {
-    public typealias Kind = BuildKeyKind
-
-    /// The kind of key
-    public let kind: Kind
-
-    /// The actual key data
-    public let key: String
-
-    public init(kind: Kind, key: String) {
-        self.kind = kind
-        self.key = key
-    }
-
-    init(key: llb_build_key_t) {
```

Comment:
This API is now a static method since it's creating specific subclasses. It's strictly an API breakage but easy to fix on client side. Is there any Swift API versioning? I see that SwiftPM uses the master branch - no version.

Reply:
This specific API isn't used by SwiftPM but the `key` getter is. That's why I added it back. Fortunately the CI is testing the build of SwiftPM as well and catched this 👍